src/components/Editor/Editor.tsx

```diff
@@ -44,7 +44,7 @@ type EditorProps = {
 const Editor = (props: EditorProps) => {
   const [isLinkEditMode, setIsLinkEditMode] = useState<boolean>(false)
   const [floatingAnchorElem, setFloatingAnchorElem] = useState<HTMLDivElement | null>(null)
-  const bg = useColorModeValue('#ffffff', '#303B4D')
+  const bg = useColorModeValue('text_area.bg_light', 'text_area.bg_dark')
```

Comment:
Avoid using hooks for color mode changes. Always use the styles first, if possible.